uniwersytet-gdanski-kulturoznawstwo-autor-rok.csl

```diff
@@ -15,8 +15,8 @@
     <category field="humanities"/>
     <category field="sociology"/>
     <category field="anthropology"/>
-    <summary>A Polish in-text citation style with wide use range for cultural studies, digital humanities. Based on Uni. of Gdansk guildelines</summary>
-    <updated>2025-08-10T17:26:29+00:00</updated>
+    <summary>A Polish in-text citation style based on Uni. of Gdansk Cultural Studies Department guidelines. Wide use range for cultural studies, digital humanities</summary>
+    <updated>2025-09-24T15:17:52+00:00</updated>
     <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
   </info>
   <locale xml:lang="pl">
```

Comment:
You can delete this whole "locale" section since the et al. term is already like this in our pl-PL locale: https://github.com/citation-style-language/locales/blob/master/locales-pl-PL.xml